manifests/server/config.pp

```diff
@@ -98,6 +98,40 @@
         value    => join($::openldap::server::ldapi_ifs, ' '),
         quoted   => 'double',
       }
+
+      if ($openldap::server::provider == 'olc') {
+        # On FreeBSD we need to bootstrap slapd.d
+        $ldif = @(EOL)
```

Comment:
That's a rather long inline file. Would you mind storing this in `files/` and sourcing it with the `file()` function?

Reply:
Ok, done.